Framework/Core/src/CommonDataProcessors.cxx

```diff
@@ -458,6 +458,13 @@ DataProcessorSpec
           // get the TableConsumer and corresponding arrow table
           auto s = pc.inputs().get<TableConsumer>(ref.spec->binding);
           auto table = s->asArrowTable();
+          if (!table->Validate().ok()) {
+            LOGP(INFO, "The table \"{}\" is not valid and will not be saved!", dh->description.str);
```

Comment:
This should be an ERROR, no?

Reply:
Writing out a message is one thing. But  how to proceed afterwards?
INFO -> continue
ERROR -> throw exception, stop ?
FATAL -> throw exception, stop ?

Reply:
ERROR does not stop, FATAL does.

Reply:
updated as suggested.